setup.py

```diff
@@ -44,10 +44,11 @@ def unique_flatten_dict(d):
 }
 
 base_extras_heavy = {
-  'umap-learn': ['umap-learn', 'dirty-cat==0.2.0', 'scikit-learn>=1.0'],
+  'umap-learn': ['umap-learn', 'dirty-cat', 'scikit-learn==1.3.2'],  
```

Comment:
forcing our exact scikit version in the universe seems bad

Reply:
yes i believe this was parked for a specific bug at 1.4, so can do <=1.3.2 for now?

Reply:
Can we not change it in this PR, and do a separate PR upgrading to scrub?